packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -1,6 +1,11 @@
-import { LitElement, html, unsafeCSS } from 'lit';
+import {
+    LitElement, html, unsafeCSS, nothing,
+} from 'lit';
+import { property } from 'lit/decorators.js';
+import { RtlMixin } from '@justeattakeaway/pie-webc-core';
 import styles from './toggle-switch.scss?inline';
-import { ToggleSwitchProps } from './defs';
+import { ToggleSwitchProps, EVENT_TOGGLE_SWITCH_CHANGED } from './defs';
+import '@justeattakeaway/pie-icons-webc/icons/IconCheck';
```

Comment:
I suspect you'll need to do this following my PR:
```suggestion
import '@justeattakeaway/pie-icons-webc/dist/icons/IconCheck.js';
```

Reply:
The path was a little different but have fixed this now 👍